internals/overlord/checkstate/handlers.go

```diff
@@ -42,6 +42,13 @@ func (m *CheckManager) doPerformCheck(task *state.Task, tomb *tombpkg.Tomb) erro
 	ticker := time.NewTicker(config.Period.Value)
 	defer ticker.Stop()
 
+	// Retrieve checkData and its channels
```

Comment:
This should now just be "channel". As is, I don't think this comment pays for itself -- should we drop it?